sport/test/football/feed/CompetitionsTest.scala

```diff
@@ -0,0 +1,80 @@
+package football.feed
+
+import java.time.{Clock, Duration, ZoneId, ZonedDateTime}
+
+import feed.Competitions
+import org.scalatest._
+import pa.FootballMatch
+import test.FootballTestData.{competitions, result, liveMatch, fixture}
+
+class CompetitionsTest extends FreeSpec with Matchers with OptionValues {
+  "Competitions" - {
+    Seq(
+      (false, Duration.ofSeconds(1), true),
```

Comment:
Would these tests be clearer if these tuples were swapped for case classes? Essentially, this would just name the fields so we can refer to them that way instead of via `_n`.

Something like:

```scala
case class TestCase(isLive: Boolean, startTimeDelta: Duration, expectedStatus: Boolean)
```

No biggie though, see what you think!

Reply:
Great suggestion, replaced the test case tuple with case class. It is indeed easier to read now 👍